ocs_ci/ocs/scale_lib.py

```diff
@@ -95,6 +96,7 @@ def create_multi_pvc_pod(
         io_runtime=None,
         pvc_size=None,
         max_pvc_size=105,
+        remove_security_context_section=False,
```

Comment:
Does false mean we are not removing it by default? let's make it default to remove the content in yaml.

Reply:
Yes. If it is False, that means that we are not removing it. Are you sure removing it from the `performance.yaml` file is ok? from my side, changing the default value to True is ok.